modules/indexer/codes/bleve.go

```diff
@@ -0,0 +1,325 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package codes
```

Comment:
Plural form of the word code is code. Could you change the name of this package to `code`?